packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -761,6 +761,25 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
   @override
   void jumpTo(double value);
 
+  /// Jumps the scrolling position from current value to delta without animation
+  /// and without checking if new value is in range, taking min/max scroll
+  /// extent into account.
+  ///
+  /// Any active animation is canceled. If the user is currently scrolling, that
+  /// action is canceled.
+  ///
+  /// If this method changes the scroll position, a sequence of start/update/end
+  /// scroll notifications will be dispatched.
+  ///
+  /// [pointerScroll] is very similar to [jumpTo], but [pointerScroll] will
+  /// update [ScrollDirection.forward] / [ScrollDirection.reverse].
+  ///
+  /// This is usually used in processing pointer signals, where delta is given
+  /// to [ScrollPosition] to calculate target value sliding.
+  // TODO(YeungKC): On macOS, a distinction should be made between mouse wheel
+  //  events and touchpad events, which can trigger an overscroll.
```

Comment:
TODOs related to trackpad scroll can refer to https://github.com/flutter/flutter/issues/23604